components/brave_wallet_ui/page/screens/send/components/account-selector/account-selector.tsx

```diff
@@ -85,6 +95,13 @@ export const AccountSelector = (props: Props) => {
         (asset?.contractAddress === "" && isFVMAccount(account)))
   }, [accounts, selectedNetwork, selectedAccountId, asset])
 
+  const { data: fvmTranslatedAddresses } = useGetFVMAddressQuery({
+    addresses: accountsByNetwork
+      .filter(account => account.accountId.coin === CoinType.ETH)
+      .map(account => account.accountId.address),
+    isMainNet: selectedNetwork?.chainId === BraveWallet.FILECOIN_MAINNET
+  })
```

Comment:
Let's do the following:
* Skip for non-FIL networks
* Create a separate memo, so case of empty addresses array is skipped.
* Skip for cases when data is not ready

Example below, but please review carefully.

```ts
const evmAddressesforFVMTranslation = React.useMemo(() =>
  accountsByNetwork
    .filter(account => account.accountId.coin === CoinType.ETH)
    .map(account => account.accountId.address),
  [accountsByNetwork])

const { data: fvmTranslatedAddresses } = useGetFVMAddressQuery(
  selectedNetwork && selectedNetwork.coin === BraveWallet.CoinType.FIL && evmAddressesforFVMTranslation.length
  ? {
      addresses: evmAddressesforFVMTranslation,
      isMainNet: selectedNetwork.chainId === BraveWallet.FILECOIN_MAINNET
    }
  : skipToken
)
```

Import `skipToken` like this:

```ts
import { skipToken } from '@reduxjs/toolkit/query/react'
```

Reply:
Thanks, in your example when we switch network from FIL to ETH fvmTranslatedAddresses doesn't reset, so i've added additional memo fvmTranslatedAddressesForNetwork which checks if current network is FIL